app/src/main/java/camp/nextstep/edu/github/MainViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package camp.nextstep.edu.github
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import camp.nextstep.edu.github.domain.model.GitHubRepos
+import camp.nextstep.edu.github.domain.usecase.GetGitHubReposUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class MainViewModel @Inject constructor(
+    private val getGitHubReposUseCase: GetGitHubReposUseCase
+) : ViewModel() {
+    private val _gitHubRepos : MutableLiveData<GitHubRepos> = MutableLiveData(GitHubRepos(emptyList()))
+    val gitHubRepos : LiveData<GitHubRepos> = _gitHubRepos
+
+    suspend fun loadGitHubRepos() {
+        _gitHubRepos.value = getGitHubReposUseCase.invoke()
+    }
+
+}
```

Comment:
presentation 레이어에서 따로 모델을 사용하시려고 했기때문에 가능하시면 이 단계에서 변환을 하셔서
뷰로 넘겨주시는 편이 좋습니다.

```suggestion
    private val _gitHubRepos : MutableLiveData<List<GitHubRepoViewData>> = MutableLiveData(emptyList())
    val gitHubRepos : LiveData<List<GitHubRepoViewData>> = _gitHubRepos

    suspend fun loadGitHubRepos() {
        _gitHubRepos.value = getGitHubReposUseCase.invoke()
            .toViewDataList()
    }
```